charts-external/communities/templates/communities.yaml

```diff
@@ -29,6 +29,11 @@ spec:
         resources: {{ .Values.resources }}
         envFrom: [{"configMapRef": {"name": "communities"}}]
         env:
+        - name: SECRET
```

Comment:
I would name this something more specific than SECRET.
SECRET has a very high probability of being overridden at some point by something else and creating a really hard to find bug.

Reply:
Fixed